README.md

```diff
@@ -55,13 +57,30 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
```

Comment:
This is really helpful, thank you! I really appreciate having some detailed steps around this process. 

However, IIRC, installing pip on macOS requires a different command and/or can be installed using Homebrew which is painless. Is it possible to confirm that these steps have been performed and validated on Linux, macOS, and Windows? If not, one option is to separate the three OSs and have individual processes for them.

Reply:
Hi @justinegeffen 
This steps are working on Windows and Linux. Yes, for Mac `HomeBrew` comes to rescue.
Nevertheless, rest of the followed commands remains unchanged.